Sources/Files.swift

```diff
@@ -485,6 +485,11 @@ public final class Folder: FileSystem.Item, FileSystemIterable {
     public var subfolders: FileSystemSequence<Folder> {
         return makeSubfolderSequence()
     }
+
+    /// A reference to the folder that is the current working directory.
+    public static var current: Folder {
+        return try! Folder(path: "")
```

Comment:
Could you make this return `FileSystem(fileManager: fileManager).currentFolder` instead? That way, any custom file manager is preserved, and we don't have to introduce another force unwrapping 🙂